packages/react-debug-tools/src/ReactDebugCustomErrors.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+/**
+ * This file contains a list of custom Errors that ReactDebugTools can throw in
+ * special occasions.
+ * The names of the errors are exported so that other packages (such as DevTools)
+ * can use them to detect and handle them separately.
+ */
+
+export const ErrorsNames = {
```

Comment:
Creating an object with keys and custom error here seems a little heavy handed/over engineering given the size/functionality of the `react-debug-tools` package.

What do you think about:

```
/* ReactDebugHooks.js */

// Note: This error needs to stay in sync with ...
// TODO: refactor this if we ever combine the devtools and debug tools packages
const error = new Error('Unsupported hook');
error.name = 'ReactDebugToolsError';
throw error;
```